l1-contracts/contracts/bridgehub/MessageRoot.sol

```diff
@@ -136,7 +157,7 @@ contract MessageRoot is IMessageRoot, Initializable {
         uint256 _chainId,
         uint256 _batchNumber,
         bytes32 _chainBatchRoot
-    ) external onlyChain(_chainId) {
+    ) external onlyAssetTracker {
```

Comment:
breaking change: when GW is updated, but other chains are not, the chains would fail to settle

Reply:
added this in assetTracker Pr